.github/dependabot.yml

```diff
@@ -3,6 +3,7 @@ updates:
   - package-ecosystem: "pip"
     directories:
```

Comment:
can you add this

    labels: [] # conflict with our labels in labels.ts

Reply:
like in typespec-azure dependabot.yml so it doesn't keep creating labels

Reply:
sounds good, let me update and re-request